codalab/lib/bundle_cli.py

```diff
@@ -1700,18 +1700,12 @@ def do_make_command(self, args):
         # Support anonymous make calls by replacing None keys with ''
         targets = [('' if key is None else key, val) for key, val in targets]
         metadata = self.get_missing_metadata(MakeBundle, args)
-        new_bundle = client.create(
-            'bundles',
-            self.derive_bundle(MakeBundle.BUNDLE_TYPE, None, targets, metadata),
-            params={'worksheet': worksheet_uuid},
-        )
 
-        # Add new bundle's location. If user specify the storage using `--store`, the bundle will be added to that storage.
+        # Add new bundle's location. If the user specifies the storage using `--store`, the bundle will be added to that storage.
         # Otherwise, the new MakeBundle will be added to default storage, which is set by the rest server.
         destination_bundle_store = metadata.get('store')
+        bundle_store_uuid = None
         if destination_bundle_store is not None:
```

Comment:
Change this to just `if destionation_bundle_store` and delete line 1710 and unindent the lines under that scope.

Reply:
Done